README.md

```diff
@@ -160,18 +162,19 @@ These instructions assume you're targeting `x64-windows`; you can change this co
 architectures.
 
 1. Install [CMake][] 3.16.5 or later, [Ninja][] 1.10.0 or later, and Visual Studio 2019 16.6 Preview 2 or later.
-2. Invoke `git clone https://github.com/microsoft/vcpkg`
-3. Invoke `cd vcpkg`
-4. Invoke `.\bootstrap-vcpkg.bat`
-5. Invoke `.\vcpkg.exe install boost-math:x64-windows` to install the boost-math dependency.
-6. Open an "x64 Native Tools Command Prompt for VS 2019".
-7. Change directories to a location where you'd like a clone of this STL repository.
-8. Invoke `git clone https://github.com/microsoft/STL`
-9. Invoke `cd STL`
-10. Invoke `cmake -G Ninja -DCMAKE_TOOLCHAIN_FILE={where your vcpkg clone is located}\scripts\buildsystems\vcpkg.cmake
+2. Change directories to a location where you'd like a clone of this STL repository.
+3. Invoke `git clone https://github.com/microsoft/STL`
+4. Invoke `cd STL`
+5. Invoke `git submodule update --init -- vcpkg/`
+6. Invoke `cd vcpkg`. Alternatively, you can use a separate installation of vcpkg e.g. `https://github.com/microsoft/vcpkg`
+7. Invoke `.\bootstrap-vcpkg.bat`
+8. Invoke `.\vcpkg.exe install boost-math:x64-windows` to install the boost-math dependency.
+9. Open an "x64 Native Tools Command Prompt for VS 2019".
+10. Change directories to the folder where this STL repository has been cloned.
+11. Invoke `cmake -G Ninja -DCMAKE_TOOLCHAIN_FILE={where your vcpkg is located}\scripts\buildsystems\vcpkg.cmake
 -S . -B {wherever you want binaries}` to configure the project. For example, `cmake -G Ninja
 -DCMAKE_TOOLCHAIN_FILE=C:\Dev\vcpkg\scripts\buildsystems\vcpkg.cmake -S . -B out\build\x64`

```

Comment:
This example still depicts a separate `vcpkg`; I think it should depict the submodule.

Reply:
Ok. Will change it to depict the submodule like DCMAKE_TOOLCHAIN_FILE=C:\STL\vcpkg\scripts\buildsystems\vcpkg.cmake